src/lucky/paginator/backend_helpers.cr

```diff
@@ -40,6 +40,47 @@ module Lucky::Paginator::BackendHelpers
     {pages, updated_query}
   end
 
+  # Call this in your actions to paginate an array.
+  #
+  # This method will return a `Lucky::Paginator` object and the requested page
+  # of items.
+  #
+  # ## Examples
+  #
+  # ```crystal
+  # class ListItems::Index < BrowserAction
+  #   get "/items" do
+  #     # The 'Array' will just show items for the requested page
+  #     # because 'paginate' will add a 'limit' and 'offset' to the query.
+  #     pages, items = paginate_array([1, 2, 3])
+  #     render IndexPage, pages: pages, items: items
+  #   end
+  # end
+  #
+  # class Users::IndexPage < MainLayout
+  #   needs pages : Lucky::Paginator
+  #   needs items : Array(T)
```

Comment:
I think this was trying to show it can be any array, but I think it may confuse people since this wouldn't compile. Using the Array(Int32) may make it clearer

```suggestion
  #   needs items : Array(Int32)
```

Reply:
Agreed, I have changed this to Int32. Thanks!